node/env0-deploy-flow.js

```diff
@@ -0,0 +1,62 @@
+const DeployUtils = require('./env0-deploy-utils');
+
+const deployUtils = new DeployUtils();
+
+const runDeployment = async (options) => {
+    await DeployUtils.init(options);
+    if (options.action === 'Deploy') {
+        await createAndDeploy(options);
+    }
+    else if (options.action === 'Destroy') {
+        await destroy(options);
+    }
+    else {
+        throw new Error(`Action ${options.action} is Invalid, Valid actions are Deploy and Destroy`);
+    }
+};
+
+const createAndDeploy = async (options) => {
+    console.log('Starting deployment');
+    let environment = await deployUtils.getEnvironment(options.environmentName, options.organizationId);
+    console.log(`got the following environment: ${JSON.stringify(environment)}`);
```

Comment:
`deployUtils.getEnvironment` already logs this, can be removed

Reply:
Done.